### PR TITLE
Open in Xcode… actions

### PR DESCRIPTION
**Pull request by pq**
Fixes: #880.

@devoncarew

### CODE REVIEW

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -252,6 +252,10 @@
               text="Flutter Packages Upgrade"
               description="Run 'flutter packages upgrade'"/>
       <separator/>
+      <action id="flutter.xcode.open" class="io.flutter.actions.OpenInXcodeAction"
+              text="Open in Xcode..."
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: Assuming the menu in IntelliJ generally use the ellipses char, we should use that here (available a few lines down).

Comment (devoncarew) [inline resources/META-INF/plugin.xml]: And - to bikeshed a bit - `Open Project in Xcode…`?

**Comment by devoncarew** [comment]
Looks good!

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: 👍

**Pull request merged**